util/aws.js

```diff
@@ -59,4 +59,13 @@ exports.mergeTags = function (tags1, tags2) {
             Value: tagsMap[tagKey]
         };
     });
+}
+
+exports.getStringArrayFromCommaSeperatedString = function (stringValue) {
+    if (!stringValue) {
+        return undefined;
+    }
+    return stringValue.split(',').map((s) => {
```

Comment:
stringValue.trim().split(/\s*,\s*/); use this then u don't need map function.

Reply:
yes done.